kuma/static/js/task-completion.js

```diff
@@ -9,13 +9,12 @@
         // asked task completion survey recently?
         var taskAskedRecently = parseInt(localStorage.getItem('taskTracker'), 10) > Date.now();
 
-        if (!ignore && !taskAskedRecently && !helpfulnessAskedRecently) {
+        if (ga && (!ignore && !taskAskedRecently && !helpfulnessAskedRecently)) {
```

Comment:
We want the survey to be available to people who have chosen to block ga so the ga check needs to only be specifically for reporting to ga and not effect other functionality.

Reply:
Ah right, I completely overlooked that in my eagerness to make the smallest change possible ;p